assets/scss/page-colours.scss

```diff
@@ -37,6 +37,7 @@ $brandings: (
 		header-link-focus: $hale-neptune-brand-colour,
 		header-link-hover-focus: $hale-neptune-header-hover-link-colour,
 		header-menu-border-colour: $hale-neptune-header-border-colour,
+    page-body-bg: $hale-neptune-shading,
```

Comment:
Does this mean the background option will only work in the neptune brand? Maybe add to uranus as well?